docs/basic-features/image-optimization.md

```diff
@@ -11,199 +11,173 @@ description: Next.js supports built-in image optimization, as well as third part
 
-## Image Component
+### Local Images
 
-To add an image to your application, import the [`next/image`](/docs/api-reference/next/image.md) component:
+You can save time by using a static `import` statement with images that live in your project:
 
 ```jsx
+import profilePic from '../public/me.png'
+```
+
+Dynamic `await import()` or `require()` are _not_ supported.
+
+The benefit to static `import`s is that several of the required props are taken care of for you. The `width`, `height`, and `blurDataURL` props will automatically be populated:
+
+```js
 import Image from 'next/image'
+import profilePic from '../public/me.png'
 
 function Home() {
   return (
     <>
       <h1>My Homepage</h1>
       <Image
-        src="/me.png"
+        src={profilePic}
         alt="Picture of the author"
-        width={500}
-        height={500}
+        // width={500} automatically provided
+        // height={500} automatically provided
+        // blurDataURL="data:..." automatically provided
+        // placeholder="blur" // Optional blur-up while loading
       />
       <p>Welcome to my homepage!</p>
     </>
   )
 }
-
-export default Home
 ```
 
-## Image Imports
-
-You can statically `import` images that live in your project. Dynamic `await import()` or `require()` are _not_ supported.
+### Remote Images
 
-With static `import`s, you only need to provide the `src` prop. The `width`, `height`, and `blurDataURL` props will automatically be populated. Alt text is still needed separately.
+For remote images, the `src` property should be set to a string containing a URL, which can be [relative](#loaders) or [absolute](#domains). Because Next.js does not have access to remote files during the build proccess, you'll need to provide the [`width`](/docs/api-reference/next/image.md#width), [`height`](/docs/api-reference/next/image.md#height) and optional [`blurDataURL`](/docs/api-reference/next/image.md#blurdataurl) props manually, as seen below.
 
-```js
+```jsx
 import Image from 'next/image'
-import profilePic from '../public/me.png'
 
 function Home() {
   return (
     <>
       <h1>My Homepage</h1>
       <Image
-        src={profilePic}
+        src="/me.png"
         alt="Picture of the author"
-        // width={500} automatically provided
-        // height={500} automatically provided
-        // blurDataURL="data:..." automatically provided
-        // placeholder="blur" // Optional blur-up while loading
+        width={500}
+        height={500}
       />
       <p>Welcome to my homepage!</p>
     </>
   )
 }
+
+export default Home
 ```
 
-For remote images, you'll need to provide the [`width`](/docs/api-reference/next/image.md#width), [`height`](/docs/api-reference/next/image.md#height) and [`blurDataURL`](/docs/api-reference/next/image.md#blurdataurl) props manually.
+>(For additional information about the sizing requirement in next/image, [see here](#image-sizing))
```

Comment:
I would consider inlining the Image Sizes section here, rather than making users click on a separate link. It feels more discoverable that way (as soon as they have the Q)

Reply:
I'm hesitant to break up the flow from remote images to loaders and domains, as you pretty much _have_ to understand what those two to use remote images. That link should be extra visible since I gave it the callout formatting, which is pretty visible on the Next.js docs styling.